apps/python/descriptions/scart.rst

```diff
@@ -69,14 +87,29 @@ Examples
       scart -I [file.mseed] [SDS archive]
       scart -I [file.mseed] --with-filecheck [SDS archive]
 
-#. Collect data from an FDSNWS server using the :ref:`global_recordstream`
-   interface and write to a miniSEED file. The data streams and the time spans are
-   defined in a list file using the option ``list``. The list can be generated e.g.
-   by :ref:`scevtstreams`.
+#. Collect data using the :ref:`global_recordstream` interface e.g. (FDSNWS server)
+   and write to a miniSEED file or import it into a local :term:`SDS` archive. The
+   data streams and the time spans can be defined in several ways. The data streams
+   (::option:list) can be automatically generated e.g. by :ref:`scevtstreams`.
 
    .. code-block:: sh
 
+      scart -I fdsnws://[server]:80 --list list.file [SDS archive]
       scart -I fdsnws://[server]:80 --list list.file --stdout > file.mseed
+      scart -I fdsnws://[server]:80 -t '[start-time]~[end-time]' --nslc list.file [SDS archive]
+      scart -I fdsnws://[server]:80 -t '[start-time]~[end-time]' -n 'NET1,NET2' [SDS archive]
```

Comment:
Although it can be difficult sometimes, I suggest to use brackets and values in examples consistently, e.g., NET1 is not a valid example, since the network code has only 2 characters and C2/C1 should have 3 characters. So you either use brackets around the placeholders or give real-life examples, e.g., N1 and CH1

With brackets, one should discriminate between optional and mandatory parameters but  found that users are often not aware of the differences and find "[]" more clear and "<>" are often mixed up with shell directives.